LICENSE

```diff
@@ -203,6 +203,23 @@ which is licensed under BSD-3-Clause, both licenses given below.
    See the License for the specific language governing permissions and
    limitations under the License.
 
+-------------------------------------------------------------------------------
+
+For tslib.
+
+file: [source code root]/packages/auth
```

Comment:
We should have this reflect the same path that you recommended:  file: [source code root]/packages/firebase

Reply:
Oops!